pandas/io/json/_json.py

```diff
@@ -31,20 +32,23 @@
 
 TABLE_SCHEMA_VERSION = "0.20.0"
 
+Serializable = Union[int, str, List, Dict]
```

Comment:
If we wanted to go crazy could subscript List and Dict recursively and with int or str, but I don't think mypy yet supports recursive types. Figure this is a good enough tradeoff